pip/pip-320.md

```diff
@@ -0,0 +1,241 @@
+# PIP-320 OpenTelemetry Scaffolding 
+
+# Background knowledge
+
+## PIP-264 - parent PIP titled "Enhanced OTel-based metric system"
+[PIP-264](https://github.com/apache/pulsar/pull/21080), which can also be viewed [here](pip-264.md), describes in high 
+level a plan to greatly enhance Pulsar metric system by replacing it with [OpenTelemetry](https://opentelemetry.io/).
+You can read in the PIP the numerous existing problems PIP-264 solves. Among them are:
+- Control which metrics to export per topic/group/namespace via the introduction of a metric filter configuration
```

Comment:
@asafm Is it supposed to look like this for the end user: `pulsar-admin topicPolicies set-exposed-metrics -m pulsar_subscriptions_count -m pulsar_producers_count persistent://a/b/c` (pseudo-CLI)

?

Reply:
Later in this PIP.

> OpenTelemetry metrics will be configured via its native OTel Java SDK configuration options

ok :)

Reply:
There will be separate file to define which metrics are filtered out. You can read about it [here](https://github.com/apache/pulsar/blob/f6515cc0f8e7dc386dc1bf579f0abfea4bdf672d/pip/pip-264.md#filtering)